priv/signs.json

```diff
@@ -6165,7 +6188,139 @@
     "source_config": [
       [
         {
-          "stop_id": "70210",
+          "stop_id": "70502",
+          "direction_id": 0,
+          "headway_direction_name": "Heath Street",
+          "routes": ["Green-E"],
+          "platform": null,
+          "terminal": false,
+          "announce_arriving": true,
+          "announce_boarding": false
+        }
+      ]
+    ]
+  },
+  {
+    "id": "lechmere_green_line_mezzanine",
+    "headway_group": "green_trunk",
+    "pa_ess_loc": "GLEC",
+    "read_loop_offset": 120,
+    "text_zone": "m",
+    "audio_zones": ["m"],
+    "type": "realtime",
+    "source_config": [
+      [
+        {
+          "stop_id": "70501",
+          "direction_id": 1,
+          "headway_direction_name": "Union Square",
+          "routes": ["Green-E"],
+          "platform": null,
+          "terminal": false,
+          "announce_arriving": true,
+          "announce_boarding": false
+        },
+        {
+          "stop_id": "70502",
+          "direction_id": 0,
+          "headway_direction_name": "Heath Street",
+          "routes": ["Green-E"],
+          "platform": null,
+          "terminal": false,
+          "announce_arriving": true,
+          "announce_boarding": false
+        }
+      ]
+    ]
+  },
+  {
+    "id": "union_sq_east",
+    "headway_group": "green_trunk",
+    "pa_ess_loc": "GUNS",
+    "read_loop_offset": 30,
+    "text_zone": "e",
+    "audio_zones": [],
+    "type": "realtime",
+    "source_config": [
+      [
+        {
+          "stop_id": "70503",
+          "direction_id": 0,
+          "headway_direction_name": "Heath Street",
+          "routes": ["Green-E"],
+          "platform": null,
+          "terminal": true,
+          "announce_arriving": false,
+          "announce_boarding": false
+        },
+        {
+          "stop_id": "70504",
+          "direction_id": 0,
+          "headway_direction_name": "Heath Street",
+          "routes": ["Green-E"],
+          "platform": null,
+          "terminal": true,
+          "announce_arriving": false,
+          "announce_boarding": false
+        }
+      ]
+    ]
+  },
+  {
+    "id": "union_sq_west",
+    "headway_group": "green_trunk",
+    "pa_ess_loc": "GUNS",
+    "read_loop_offset": 30,
+    "text_zone": "w",
+    "audio_zones": ["e", "w", "m"],
```

Comment:
Any particular reason to attach the audio zones to the "west" config? Not that there's any functional difference, but I might have expected it to be either the first Union Square config in the file (east), or the "middle" one (mezzanine).

Reply:
Mainly just because all of the predictions that we'd be showing for this station would be "westbound", but that's a pretty loose reasoning. But it does make a little bit more sense to me to put it with the mezzanine zone now that I think about it